substrate/frame/support/src/lib.rs

```diff
@@ -508,6 +508,27 @@ pub use frame_support_procedural::{
 	construct_runtime, match_and_insert, transactional, PalletError, RuntimeDebugNoBound,
 };
 
+/// Construct a runtime, with the given name and the given pallets.
+///
+/// # Example:
+#[doc = docify::embed!("src/tests/runtime.rs", runtime_macro)]
+///
+/// # Legacy Ordering
```

Comment:
Having a list of all the attributes would be nice.

Reply:
Currently, only `legacy_ordering` is supported as an attribute but will restructure it as a list for future extensions.